techmatters/flex-plugins
# Deploy to all staging/production helplines

## Description
3 new workflows:
- Deploy Webchat to all Staging Helplines
- Deploy Webchat to all Production Helplines
- Deploy Flex to all Production Helplines

Added forms definitions for helpline F2 (which we should reevaluate if we should delete this account).

~TODO: Turns out there's a lot of repeated github code in here. I'll make it more DRY~

It uses a similar system for DRYing workflows as `serverless` (with small differences)

### Related Issues
Fixes [CHI-1756](https://tech-matters.atlassian.net/browse/CHI-1756) and [CHI-1758](https://tech-matters.atlassian.net/browse/CHI-1758)

### Verification steps
After merged, check that both GitHub Actions show up and they work as expected

[CHI-1756]: https://tech-matters.atlassian.net/browse/CHI-1756?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CHI-1758]: https://tech-matters.atlassian.net/browse/CHI-1758?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
